pkg/backup/backup_pv_action.go

```diff
@@ -24,28 +24,29 @@ import (
 
 	v1 "github.com/heptio/velero/pkg/apis/velero/v1"
 	"github.com/heptio/velero/pkg/kuberesource"
+	"github.com/heptio/velero/pkg/plugin/velero"
 )
 
-// backupPVAction inspects a PersistentVolumeClaim for the PersistentVolume
+// PVAction inspects a PersistentVolumeClaim for the PersistentVolume
 // that it references and backs it up
-type backupPVAction struct {
+type PVAction struct {
```

Comment:
this should either stay as `BackupPVAction`, or become `PVCAction`.  Note that it operates on *PVCs*, and returns the claimed PV as an additional item to back up.  Hence "Backup PV"